fhir-persistence-cos/src/main/java/com/ibm/fhir/persistence/cos/client/COSPropertiesAdapter.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.persistence.cos.client;
+
+import java.util.Properties;
+
+/**
+ * Adapter to support reading of COS properties from a {@link Properties} instance
+ */
+public class COSPropertiesAdapter {
```

Comment:
I commented on these names.  I'd really like to have this unified with the bulkdata constant names.